posthog/session_recordings/queries/session_recording_list_from_replay_summary.py

```diff
@@ -85,8 +85,12 @@ def __init__(
     SELECT distinct log_source_id as session_id
     FROM log_entries
     PREWHERE team_id = %(team_id)s
-            AND timestamp >= %(clamped_to_storage_ttl)s
-            AND timestamp <= now()
+            -- regardless of what other filters are applied
+            -- limit by storage TTL
+            AND e.timestamp >= %(clamped_to_storage_ttl)s
+            -- make sure we don't get the occasional unexpected future event
+            AND e.timestamp <= now()
+            -- and then any time filter for the events query
```

Comment:
copied the comments up here to make the duplication more obvious so it'll annoy me and I can extract it next time I'm changing this file